wazo_confd/plugins/user/api_v2_0.yml

```diff
@@ -0,0 +1,313 @@
+          readOnly: true
+    - $ref: '#/definitions/UserRelationAgent'
+    - $ref: '#/definitions/UserRelationFallbacks'
+    - $ref: '#/definitions/UserRelationForwards'
+    - $ref: '#/definitions/UserRelationGroups'
+    - $ref: '#/definitions/UserRelationIncalls'
+    - $ref: '#/definitions/UserRelationLines'
+    - $ref: '#/definitions/UserRelationServices'
+    - $ref: '#/definitions/UserRelationSwitchboards'
+    - $ref: '#/definitions/UserRelationVoicemail'
+    - $ref: '#/definitions/UserRelationQueues'
+    - $ref: '#/definitions/UserRelationCallPickupTargets'
+    - required:
+      - firstname
+  UserRelationBase:
+    properties:
+      uuid:
+        type: string
+        description: User UUID. This ID is globally unique across multiple Wazo instances
+        readOnly: true
+      firstname:
+        type: string
+        description: User firstname
+      lastname:
+        type: string
+        description: User lastname
+  UserRelationGroups:
+    properties:
+      groups:
+        type: array
+        readOnly: true
+        items:
+          $ref: '#/definitions/GroupRelationBase'
+  UserRelationIncalls:
+    properties:
+      incalls:
+        type: array
+        readOnly: true
+        items:
+          $ref: '#/definitions/UserRelationIncall'
+  UserRelationIncall:
+    allOf:
+      - $ref: '#/definitions/IncallRelationBase'
+      - $ref: '#/definitions/IncallRelationExtensions'
+  UserRelationLines:
+    properties:
+      lines:
+        type: array
+        readOnly: true
+        items:
+          $ref: '#/definitions/UserRelationLine'
+  UserRelationLine:
+    allOf:
+      - $ref: '#/definitions/LineRelationBase'
+      - $ref: '#/definitions/LineRelationEndpoints'
+      - $ref: '#/definitions/LineRelationExtensions'
+  UserRelationFallbacks:
+    properties:
+      fallbacks:
+        readOnly: true
+        $ref: '#/definitions/UserFallbacks'
+  UserRelationForwards:
+    properties:
+      forwards:
+        readOnly: true
+        $ref: '#/definitions/UserForwards'
+  UserRelationServices:
+    properties:
+      services:
+        readOnly: true
+        $ref: '#/definitions/UserServices'
+  UserRelationSwitchboards:
+    properties:
+      switchboards:
+        type: array
+        readOnly: true
+        items:
+          $ref: '#/definitions/SwitchboardRelationBase'
+  UserRelationVoicemail:
+    properties:
+      voicemail:
+        readOnly: true
+        $ref: '#/definitions/VoicemailRelationBase'
+  UserRelationQueues:
+    properties:
+      queues:
+        type: array
+        readOnly: true
+        items:
+          $ref: '#/definitions/QueueRelationBase'
+  UserRelationAgent:
+    properties:
+      agent:
+        readOnly: true
+        $ref: '#/definitions/AgentRelationBase'
+  UserRequest:
+    title: Unified User
+    properties:
+        user:
+          type: object
+          description: User Section
+          $ref: '#/definitions/User'
```

Comment:
IMHO this is incompatible with the `/2.0/users` pattern: if the resource is a user, then the fields of this resource should be first-level, e.g. `{firstname: ..., lines: {...}}`

For me `/2.0/...` is the future of the confd API, and all resources will eventually move to `/2.0/...`, so it needs to be ready for more resources.

Having `users: {...}` would be correct for a URL like `/unified/users` that exists *besides* `/users`, but not for `/2.0/users`: either we'll have to keep the same pattern for other resources (e.g. `/2.0/queues` will also have a `queues: {...}` field), or we will break homogeneity between all resources.

And if we remove the `user` key, then do we really need `/2.0`, i.e. just add new keys in `POST /1.1/users`?